spec/fixtures/custom_plugins/kong/plugins/rpc-debug/handler.lua

```diff
@@ -0,0 +1,76 @@
+local kong_meta = require("kong.meta")
+
+local _M = {
+  PRIORITY = 1000,
+  VERSION = kong_meta.version,
+}
+
+local original_callbacks = {}
+local inc_id = 0
+
+function _M.init_worker()
+  kong.rpc.callbacks:register("kong.rpc.debug.register", function(node_id, register_payload)
```

Comment:
Should we rename it to `rpc-testing`? It dose not look like `debug`.

Reply:
I can be used for debug purpose and the connected client act as a debugger